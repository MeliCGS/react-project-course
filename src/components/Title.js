function Title({children}) {
    return (
        <div>
            <h2 className="text-intro">{children}</h2>
        </div>
    )
}

export default Title