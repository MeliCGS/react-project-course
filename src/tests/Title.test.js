import React from 'react'
import '@testing-library/jest-dom/extend-expect'
import '@testing-library/jest-dom'
import { render } from '@testing-library/react'
import Title from '../components/Title'

describe('Pruebas en el componente Title', () => {
    test('El componente deberia renderizar la etiqueta <h2 />', () => {
        const { container } = render(<Title />)
        expect(container).toMatchSnapshot()
    })
})