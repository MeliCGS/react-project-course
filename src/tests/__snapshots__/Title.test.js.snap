// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el componente Title El componente deberia renderizar la etiqueta <h2 /> 1`] = `
<div>
  <div>
    <h2
      class="text-intro"
    />
  </div>
</div>
`;
